app/models/order.rb

```diff
@@ -0,0 +1,35 @@
+class Order < ApplicationRecord
+  ### CONSTANTS ###
+
+  STATUSES = {
+    cart: 0,
+    received: 1,
+    cancelled: 2,
+    ready: 3,
+    'picked up': 4
+  }
+
+  enum status: STATUSES
+
+  ### ASSOCIATIONS ###
+
+  belongs_to :user
+  belongs_to :branch
+  has_many :line_items, autosave: true, dependent: :delete_all
+  has_many :meals, through: :line_items
+  has_many :ingredients_meals, through: :meals
+
+  def add_meal(meal_id)
```

Comment:
Shouldn't we check here, whether the item is really available and active in this branch?

Reply:
Done